examples/equilibria/coilset_opt_problem_tutorial.py

```diff
@@ -145,24 +169,124 @@
 
 # %%
 
-magnetic_targets, magnetic_core_targets = double_null_ST.init_targets()
+x_lcfs = np.array([1.0, 1.67, 4.0, 1.73])
+z_lcfs = np.array([0, 4.19, 0, -4.19])
+
+lcfs_isoflux = IsofluxConstraint(
+    x_lcfs, z_lcfs, ref_x=x_lcfs[2], ref_z=z_lcfs[2], constraint_value=0.1
+)
+
+x_lfs = np.array([1.86, 2.24, 2.53, 2.90, 3.43, 4.28, 5.80, 6.70])
+z_lfs = np.array([4.80, 5.38, 5.84, 6.24, 6.60, 6.76, 6.71, 6.71])
+x_hfs = np.array([1.42, 1.06, 0.81, 0.67, 0.62, 0.62, 0.64, 0.60])
+z_hfs = np.array([4.80, 5.09, 5.38, 5.72, 6.01, 6.65, 6.82, 7.34])
+
+x_legs = np.concatenate([x_lfs, x_lfs, x_hfs, x_hfs])
+z_legs = np.concatenate([z_lfs, -z_lfs, z_hfs, -z_hfs])
+
+legs_isoflux = IsofluxConstraint(
+    x_legs, z_legs, ref_x=x_lcfs[2], ref_z=z_lcfs[2], constraint_value=0.1
+)
+
+magnetic_targets = MagneticConstraintSet([lcfs_isoflux, legs_isoflux])
+magnetic_core_targets = MagneticConstraintSet([lcfs_isoflux])
 
 # %%[markdown]
 
 # We also specify an initial Equilibrium state to be used in the optimisation.
 
+# For this we need a Grid and some plasma profiles
+
 # %%
 
-grid = double_null_ST.init_grid()
-profile = double_null_ST.init_profile()
+# Intialise some parameters
+R_0 = 2.6
+Z_0 = 0
+B_t = 1.9
+I_p = 16e6
+
+r0, r1 = 0.2, 8
+z0, z1 = -8, 8
+nx, nz = 129, 257
+grid = Grid(r0, r1, z0, z1, nx, nz)
+
+pprime = np.array(
```

Comment:
Worth having some data in examples/equilbria to pull this from? I think the long arrays of floats used to set up the initial state and constraints hurts readability of the example here